pkg/distributor/distributor_test.go

```diff
@@ -906,15 +785,6 @@ func TestShardCountFor(t *testing.T) {
 			wantShards:     1,
 			wantErr:        false,
 		},
-		{
```

Comment:
I removed this scenario because it is impossible to have `streamSize=2` if number of entries is 0.